src/app/series/directives/series-basic.component.spec.ts

```diff
@@ -10,7 +10,7 @@ describe('SeriesBasicComponent', () => {
 
   cit('shows the basic series edit fields', (fix, el, comp) => {
     expect(el).not.toQuery('publish-fancy-field');
-    comp.series = {};
+    comp.series = {changed: () => false};
     fix.detectChanges();
 
     expect(el.queryAll(By.css('publish-fancy-field')).length).toEqual(3);
```

Comment:
teeny quibbly thing: finding this phrasing a little misleading since this test all together makes it seem like 'full description' is among the `publish-fancy-field`s